string name
float64[] goal_pose
float64[] weights_updated
float64[] means_updated
float64[] variances_updated
